modules/crawler.py

```diff
@@ -330,47 +284,39 @@ def get_techstack_indicators(domains: List[str], api_key: str):
         api_result = requests.get(tech_stack_query)
         data = json.loads(api_result.content)
         # API supports querying multiple sites at a time, hence the embedded structure
+        technologies = []
         for result_item in data["Results"]:
             result = result_item["Result"]
-            tech_stack = []
             for path in result["Paths"]:
-                technologies = [
-                    {
-                        "indicator_type": "techstack",
-                        "indicator_content": {
+                technologies.extend([
+                    Indicator("techstack",{
                             "name": tech.get("Name"),
                             "link": tech.get("Link"),
                             "tag": tech.get("Tag"),
                             "subdomain": path["SubDomain"],
-                        },
-                        "domain_name": domain,
-                    }
+                        })
                     for tech in path["Technologies"]
-                ]
-                tech_stack.extend(technologies)
```

Comment:
another situation of breaking out individual items into their own indicators rather than one indicator with a giant list.